assets/src/edit-story/components/canvas/framesLayer.js

```diff
@@ -44,7 +44,7 @@ const FramesPageArea = withOverlay(
 
 const FrameSidebar = styled.div`
   position: absolute;
-  left: -200px;
+  left: -205px;
```

Comment:
This is pretty magic - but it's added the offset of the background outline (as to not partially cover it). It should probably use a constant as well coming from that shared helper.